announcements/2020-Week-48/announcement.md

```diff
@@ -0,0 +1,18 @@
+# :zap: This Week in Provider-Independent Announcements (November 27, 2020):zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed Apps
+
+1. [Grafana v0.1.0-alpha1](https://github.com/giantswarm/grafana-app) is now available in the Giant Swarm Catalog. It contains the latest upstream Grafana v7.3.1.
+2. [Cert Manager v2.3.3](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#233---2020-11-23) schedules hook Jobs on master nodes.
+3. [NGINX IC v1.11.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#1110---2020-11-18) updates image to v0.41.2 and adds ability to extend the app with specific values from appcatalog.
```

Comment:
@giantswarm/app-squad-nginx can you help me - how does it make customer's life better (if any)? 😀

Reply:
no real customer-facing changes here, aside from the version bump which fixes some bugs.

Reply:
ah ok! better to not include it? wdyt?

Reply:
I will remove it.